lms/djangoapps/instructor/management/commands/collect_course_forums_data.py

```diff
@@ -0,0 +1,51 @@
+"""
+Command to retrieve aggregated course forums data in a .csv
+"""
+import csv
+import optparse
+import os
+
+from instructor.utils import collect_course_forums_data
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+from instructor.management.commands.collect_ora2_data import preprocess
+
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    """
+    Query aggregated open assessment data, write to .csv
+    """
+
+    help = ("Usage: collect_course_forums_data <course_id> --output-dir=<output_dir>")
+    args = "<course_id>"
+    option_list = BaseCommand.option_list + (
+        optparse.make_option('-o', '--output-dir',
+                             action='store', dest='output_dir', default=None,
+                             help="Write output to a directory rather than stdout"),
+    )
+
+    def handle(self, *args, **options):
+        if not args:
+            raise CommandError("Course ID must be specified to fetch data")
+
+        try:
+            course_id = SlashSeparatedCourseKey.from_deprecated_string(args[0].decode('utf-8'))
```

Comment:
was going off of https://github.com/Stanford-Online/edx-platform/blob/master/lms/djangoapps/instructor/management/commands/collect_ora2_data.py#L32